po/zh-TW.po

```diff
@@ -21350,3 +21455,209 @@ msgid ""
 msgstr ""
 "根據預設，閉包會依據參照來擷取內容 (如果可行的話)。`move` 關鍵字則可讓閉包根"
 "據值 來擷取內容。"
+
+#: src/welcome-day-2.md:10
+msgid "Structs and methods."
+msgstr "結構體和方法。"
+
+#: src/memory-management/stack.md:1
+msgid "Stack and Heap Example"
+msgstr "堆疊和堆積範例"
+
+#: src/ownership/move-semantics.md:23
+msgid ""
+"It is only the ownership that moves. Whether any machine code is generated "
+"to manipulate the data itself is a matter of optimization, and such copies "
+"are aggressively optimized away."
+msgstr ""
+"只有擁有權才會轉移。無論是否產生任何機器碼來操控資料本身，這都是最佳化的相關"
+"問題，系統會主動將這些副本最佳化。"
```

Comment:
how about “是否產生任何機器碼來操控資料本身是一個最優化問題" ？

Reply:
Just in case you haven't seen it: GitHub has a suggestion feature. It's the left-most button above and it looks like this when used:

```suggestion
"是否產生任何機器碼來操控資料本身是一個最優化問題"
```

It's likely wrong in this case since your suggestion might have spanned both lines? If so, you can highlight both lines when you do a review and make a multi-line suggestion.

The advantage of the suggestion is that the PR author can apply them with a few clicks. It saves a lot of round-trips :smile:

Reply:
Thank you! That's good to know!